File: src/main/java/com/jcloisterzone/figure/neutral/Donkey.java
package com.jcloisterzone.figure.neutral;

import com.jcloisterzone.Immutable;
import com.jcloisterzone.board.pointer.BoardPointer;

@Immutable
public class Donkey extends NeutralFigure<BoardPointer> {

    private static final long serialVersionUID = 1L;

    public Donkey(String id) {
        super(id);
    }
}

File: src/main/ts/com/jcloisterzone/figure/neutral/Donkey.ts
import type { BoardPointer } from "../../board/pointer/BoardPointer.js";
import { NeutralFigure } from "./NeutralFigure.js";

/** The Donkey (Under the Big Top). */
export class Donkey extends NeutralFigure<BoardPointer> {
  static readonly simpleName = "Donkey";
  constructor(id: string) {
    super(id);
  }
}
